examples/mirror_actors.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+
+import numpy as np
+from myterial import orange
+from rich import print
+
+from brainrender import Scene
+from brainrender._io import load_mesh_from_file
+from brainrender.actor import Actor
+from brainrender.actors import Neuron, make_neurons, Points
```

Comment:
I don't think this is used

```suggestion
from brainrender.actors import Neuron, Points
```

Reply:
Great catch!